docker-app/qfieldcloud/subscription/admin.py

```diff
@@ -117,11 +117,14 @@ class SubscriptionAdmin(QFieldCloudModelAdmin):
     form = SubscriptionModelForm
 
     fields = (
-        "plan",
+        "regular_plan",
+        "trial_plan",
         "account",
         "status",
         "active_since",
         "active_until",
+        "is_trialing_display",
```

Comment:
```suggestion
        "is_trialing_display",
```

Can't we just use `is_trialing` without a custom rendering function?

Reply:
Yes, as it is flagged as a property.